arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentCollectionViewController.swift

```diff
@@ -116,15 +77,8 @@ class ContentCollectionViewController: UIViewController, UICollectionViewDataSou
     
```

Comment:
Could you remove this TODO? If this is something we'd like to implement, this belongs as an issue and out of version control.

I recognize this is not a part of the original PR, I just noticed it.

Reply:
I took a look at the TODO and I am all for removing it, but I think it should be a separate PR. Removing it could be done before or after this PR is merged without affecting anything. That's kind of my rule of thumb for making unrelated changes in a PR.